docs/ref/paginator.txt

```diff
@@ -161,9 +161,80 @@ Attributes
 
     A 1-based range iterator of page numbers, e.g. yielding ``[1, 2, 3, 4]``.
 
+``AsyncPaginator`` class
+========================
+
+.. versionadded:: 5.2
+
+.. class:: AsyncPaginator(object_list, per_page, orphans=0, allow_empty_first_page=True, error_messages=None)
+
+    Asynchronous version of :class:`Paginator`.
+
+.. attribute:: AsyncPaginator.object_list
+
+    See :attr:`.Paginator.object_list`.
+
+.. attribute:: AsyncPaginator.per_page
+
+    See :attr:`.Paginator.per_page`.
+
+.. attribute:: AsyncPaginator.orphans
+
+    See :attr:`.Paginator.orphans`.
+
+.. attribute:: AsyncPaginator.allow_empty_first_page
+
+    See :attr:`.Paginator.allow_empty_first_page`.
+
+.. attribute:: AsyncPaginator.error_messages
+
+    See :attr:`.Paginator.error_messages`.
+
+Methods
+-------
+
+.. method:: AsyncPaginator.aget_page(number)
+
+    See :meth:`.Paginator.get_page`.
+
+.. method:: AsyncPaginator.apage(number)
+
+    See :meth:`.Paginator.page`.
+
+.. method:: AsyncPaginator.aget_elided_page_range(number, *, on_each_side=3, on_ends=2)
+
+    See :meth:`.Paginator.get_elided_page_range`.
+
+.. method:: AsyncPaginator.acount()
+
+    The asynchronous version is supported as a method.
+
+    See :attr:`.Paginator.count`.
+
+.. method:: AsyncPaginator.anum_pages()
+
+    The asynchronous version is supported as a method.
+
+    See :attr:`.Paginator.num_pages`.
+
+.. method:: AsyncPaginator.apage_range()
+
+    The asynchronous version is supported as a method.
+
+    See :attr:`.Paginator.page_range`.
+
+Attributes
+----------
+
+.. attribute:: AsyncPaginator.ELLIPSIS
+
+    See :attr:`.Paginator.ELLIPSIS`.
+
 ``Page`` class
 ==============
 
+.. versionadded:: 5.2
```

Comment:
I think this annotation was meant to go elsewhere, maybe to ELLIPSIS or AsyncPage?